release/release.go

```diff
@@ -108,17 +110,34 @@ func readOSRelease() OS {
 	return osRelease
 }
 
+var ioutilReadFile = ioutil.ReadFile
+
+func isWSL() bool {
+	version, err := ioutilReadFile("/proc/version")
```

Comment:
Is this documented somehow by Microsoft? I also noticed that WSL has a particular uname output that could be used as an alternative detection strategy.

Reply:
https://github.com/Microsoft/WSL/issues/423#issuecomment-221627364

And, AIUI they themselves have code that checks in this way.